pkg/virt-api/webhooks/validating-webhook/admitters/vmpool-admitter.go

```diff
@@ -41,7 +41,8 @@ import (
 )
 
 type VMPoolAdmitter struct {
-	ClusterConfig *virtconfig.ClusterConfig
+	ClusterConfig           *virtconfig.ClusterConfig
+	KubeVirtServiceAccounts map[string]struct{}
```

Comment:
Why `map[string]struct{}` and not `map[string]string`?

Reply:
The map contain keys which are KubeVirt's ServiceAccount full names (including the namespace KubeVirt is deployed to).
We only care if the map contains the key - thus no need to have a value.